cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -207,12 +242,13 @@ private AuthenticationClient getAuthenticationClient(ConnectionConfig connection
     return authenticationClient;
   }
 
-  private AccessToken getSavedAccessToken(String hostname) {
+  private UserAccessToken getSavedAccessToken(String hostname) {
     File file = getAccessTokenFile(hostname);
     if (file.exists() && file.canRead()) {
```

Comment:
Shouldn't need this check. Simply try to read from the file and it'll throw IOException if the file is not readable.